### DISCUSSION TITLE
Which version of NTLM does Insomnia support?

### DISCUSSION THREAD

**Discussion by haydenhancock** [Kong/insomnia#3097] [Help]
Does anyone know which version of NTLM Insomnia supports?

**Comment by nijikokun** [comment]
Since we leverage `libcurl` to do `NTLM` authentication, looks like it's reliant on the [CURL version](https://github.com/JCMais/node-libcurl/blob/c8056092017bdbcbfc1ec423a6654d058792d32c/lib/enum/CurlVersion.ts#L38). Can you make the same request through CURL? 

Note: Insomnia only supports [`NTLM`](https://github.com/Kong/insomnia/blob/021b9f581009eeb5f913991634380f23d8df123a/packages/insomnia-app/app/models/request.js#L135) not `NTLM_WB`.
